src/events/music/musicSongVolumeUpdate.ts

```diff
@@ -11,8 +11,15 @@ export default class extends Event {
 			const { language } = channel.guild;
 			const response =
```

Comment:
this can move into a sendlocale with a ternary inside it (even tho it's gonna look hacky as hell)

Reply:
I rather prefer this structure tbh